### PR TITLE
Added support for /location

### PR DESCRIPTION
**Pull request by Aztherion**
Visma.net version 6.13 saw the release of /location, enabling developers to get/create/update customer locations.

The implementation differs slightly compared to other endpoints as /location in almost all circumstances requires a baccountId. 

eg. 
GET /location/{baccountid}
returns a range of locations for a BAccount. 

GET /location/{baccountid}/{locationid}
Returns a specific location of a BAccount.

### CODE REVIEW

File: Visma.net/lib/VismaNetControllers.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ internal static class VismaNetControllers
         public const string Branch = "controller/api/v1/branch";
 		public const string Attachment = "controller/api/v1/attachment";
         public const string Warehouse = "controller/api/v1/warehouse";
+        public const string Locations = "controller/api/v1/location";
```
Comment (omelhus) [inline Visma.net/lib/VismaNetControllers.cs]: Rename to Location

File: Visma.net/lib/DAta/LocationData.cs [hunk 2]
```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+using ONIT.VismaNetApi.Models;
+
+namespace ONIT.VismaNetApi.Lib.Data
+{
+    public class LocationData : BaseCrudDataClass<Location>
```
Comment (omelhus) [inline Visma.net/lib/DAta/LocationData.cs]: How about just skipping inheriting from BaseCrudDataClass?

Comment (Aztherion) [reply Visma.net/lib/DAta/LocationData.cs]: Would makes sense, though the implementation makes use of Add and Update in BaseCrudDataClass and we might want to keep the inheritance for code consistency but I'll let you decide on that.

File: Visma.net/lib/VismaNetControllers.cs (hunk 1 above)
Comment (Aztherion) [reply Visma.net/lib/VismaNetControllers.cs]: Got it.

File: Visma.net/lib/DAta/LocationData.cs (hunk 2 above)
Comment (Aztherion) [reply Visma.net/lib/DAta/LocationData.cs]: Has a second look at it and changed the inheritance from BaseCrudDataClass to BaseDataClass.

**Comment by Aztherion** [comment]
Issues taken care of and the pull request has been changed from master to dev. Anything else, or are we good to go?

**Pull request merged**